libs/inspecjs/src/compat_impl/compat_inspec_1_0.ts

```diff
@@ -47,7 +47,11 @@ abstract class HDFControl10 implements HDFControl {
     this.waived = waived;
     this.isProfile = isProfile;
 
-    this.rawNistTags = HDFControl10.compute_raw_nist_tags(this.wraps);
+    const tempNistTags = HDFControl10.compute_raw_nist_tags(this.wraps);
+
+    this.rawNistTags = Array.isArray(tempNistTags)
+      ? tempNistTags
+      : [tempNistTags];
```

Comment:
May want to comment here and stated that rawNisTags must be an array (and possible the reason why)